src/main/java/software/amazon/cloudwatchlogs/emf/model/MetricDirective.java

```diff
@@ -67,14 +67,22 @@ void putDimensionSet(DimensionSet dimensionSet) {
     }
 
     void putMetric(String key, double value) {
-        putMetric(key, value, Unit.NONE);
+        putMetric(key, value, Unit.NONE, StorageResolution.STANDARD);
     }
 
     void putMetric(String key, double value, Unit unit) {
+        putMetric(key, value, unit, StorageResolution.STANDARD);
+    }
+
+    void putMetric(String key, double value, StorageResolution storageResolution) {
```

Comment:
Also never used

Reply:
Please take a look at [this](https://github.com/awslabs/aws-embedded-metrics-java/pull/129/files#r1054936457) comment.

Reply:
These methods are used to help with tests. Added a comment to specify that these are helper methods.